wiki/markdown/configuration.md

```diff
@@ -45,5 +49,111 @@ URI schemes that are accepted are 'file','http' and 'https'.
+to configure the collection name, the time to live and the name of the
+aggregated object using the below properties:
+
+* aggregated.object.name
+* aggregated.collection.name
+
+If Eiffel Intelligence is set up with [all_event_rules](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/main/resources/all_event_rules.json)
+it is recommended to set a time to live value to avoid having a copy of
+Event repository. Recommended settings is 10 minutes.
+
+* aggregated.collection.ttlValue (*seconds*)
+
+When performing queries towards an aggregated object, **search.query.prefix**
+is used to access the aggregated object when using JMESPath expressions.
+
+Consider the following query:
+
+    {
+      "criteria": {
+        "object.identity":"pkg:maven/com.mycompany.myproduct/sub-system@1.1.0"
+      }
+    }
+
+Eiffel Intelligence will replace the prefix "object." with the name of the aggregated
+object (in this case "aggregatedObject"), which is defined by **aggregated.object.name**.
+Read more examples about [queries here](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/query.md).
+
+
+### Testing aggregation rules
+
+To test new rules for Eiffel Intelligence the property **testaggregated.enabled**
+can be set to true. This gives users the possibility to try out different rule sets
+on a specific set of Eiffel events and see the resulting aggregated object.
+
+
+### Wait list
+
+If Eiffel Intelligence receives events that are not connected to any
+existing aggregated objects, and it is not declared as start event in the
+rules, the event is stored in the wait list database for a while. The
+"time-to-live" value limits how long an event is stored in the wait list.
+
+* waitlist.collection.name
+* waitlist.collection.ttlValue (*seconds*)
+
+It is possible to configure how often Eiffel Intelligence should handle
+unprocessed events lying in the wait list. These properties are in **milliseconds**:
+
+* waitlist.initialDelayResend (*milliseconds*)
+* waitlist.fixedRateResend (*milliseconds*)
+
+
+## Subscriptions
+
+Eiffel Intelligence stores subscriptions in a database with the collection
+name configured with the property **subscription.collection.name**. When
+subscriptions have fulfilled their requirements and notifications
+should not be repeated, the subscription is stored together with the
+matched aggregated object in the database. The name of this particular
+collection is defined by the below property:
+
+* subscription.collection.name
+* subscription.collection.repeatFlagHandlerName
+
+
+### Notifications
+
+**email.sender** defines who should be the sender of the email Eiffel
+Intelligence sends when a subscription is triggered.
+The value for email.subject is used by all subscriptions which have been
+created via the REST API. When a subscription is created using the Eiffel
+Intelligence front-end GUI it is possible to set individual email subject
+for each subscription.
+
+* email.sender
```

Comment:
Whos mail address is sender? The receiver mail address or the mail address that will been seen in the mail it was sent from?

Reply:
Okay, I'll describe it clearer